doc/openapi/v0/cyphernode-api.yaml

```diff
@@ -350,6 +350,135 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /get_txns_by_watchlabel/{label}/{count}:
+    get:
+      parameters:
+        - in: "path"
+          name: "label"
+          description: "Xpub label"
+          required: true
+          schema:
+            type: "string"
+        - in: "path"
+          name: "count"
+          description: "Number of transactions to return"
+          required: false
+          schema:
+            type: "number"
+      tags:
+        - "watching transactions"
+        - "core features"
+      summary: "Get list of transactions observed for addreses belonging to this label"
+      description: "Get list of transactions observed for addreses belonging to this label"
+      operationId: "get_txns_by_watchlabel"
+      responses:
+        '200':
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - "label_txns"
+                properties:
+                  watches:
+                    type: "array"
+                    items:
+                      $ref: '#/components/schemas/WatchXPubTxn'
+        '403':
+          $ref: '#/components/schemas/ApiResponseNotAllowed'
+        '503':
+          description: "Resource temporarily unavailable"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponseTemporarilyUnavailable'
+  /get_txns_spending/{count}/{skip}:
```

Comment:
Im not a fan of this notation /get_txns_spending/{count}/{skip}.
1) This will prevent implicit defaults chosen by the proxy, cause count and skip is not optional
2) This will make it impossible to get ALL txs unless u know how many there are which might involve a second API call

I would prefer to put the required parameters into the path and the optional parameters into the query string.
Depending on the resources needed to respond to the request the proxy can choose to return ALL txs unless otherwise specified or if its resource intensive choose a good default for skip and count

Reply:
1. Count and skip are optional in both URLS and have fallback defaults set in the fn when non are passed  (Count =10, Skip =0)
2. You are right about not being able to get ALL txns, in my opinion though the majority of use cases will not be to get all txns. If you want to get a large amount of txn that you can easily set that with a high count and skip and paginate your results

Reply:
Yes I know setting count to 99999 is possible but hacky :)